static/css/style.css

```diff
@@ -45,6 +58,117 @@ form {
       background-color: #b07337;
       color: #000; }
 
+.map {
+  padding-left: 0px;
+  padding-right: 0px; }
+
+.map iframe {
+  padding-right: 0px; }
+
+.footer-top .row {
+  background: #51a3d6;
+  margin-top: 40px;
+  padding-right: 0px; }
+
+.col-12 {
+  padding-right: 0px;
+  padding-left: 0px; }
+
+.col-lg-3 {
+  margin-top: 60px;
+  color: #fff; }
```

Comment:
We have a variable for #fff set it base.scss :) 
We should use variables as possible so it's easierto make changes.
In this case, color: $primary-bg;